packages/datepickers/stories/4-Range.stories.tsx

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useMemo, useState } from 'react';
+import styled from 'styled-components';
+import { Meta, Story } from '@storybook/react';
+import { DatepickerRange } from '@zendeskgarden/react-datepickers';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Field, Label, Input, Message } from '@zendeskgarden/react-forms';
+import { addDays, compareAsc } from 'date-fns';
+
+export default {
+  title: 'Components/Datepicker/DatepickerRange',
```

Comment:
Can we get all of these tucked under a single `Datepickers` folder?

Reply:
This was unintentionally caused by a rebase. Everything is now under `Components/Datepickers`